### DISCUSSION TITLE
Precompile route(s) in dev

### DISCUSSION THREAD

**Discussion by elado** [vercel/next.js#24840] [Ideas]
Many times I start dev server (`yarn dev`) and go do something else while it starts. Then, when I come back and access the server `http://localhost:3000/foo` it starts building the route and I get to wait a little more.

Would be great if some routes could start async compilation as I start the server, so I don't have to wait for them on access.

Does that already exist and I missed it?

**Comment by bobarros** [comment]
A long conversation about this here --> https://github.com/vercel/next.js/issues/6554

**Reply by elado** [reply]
My workaround: https://github.com/vercel/next.js/issues/6554#issuecomment-833951620
